deployments/liqo/values.yaml

```
@@ -34,7 +34,7 @@ controllerManager:
     # -- the threshold (in percentage) of resources quantity variation which triggers a ResourceOffer update.
     offerUpdateThresholdPercentage: ""
     # -- The address of an external resource monitor service, overriding the default resource computation logic based on the percentage of available resources. Leave it empty to use the standard local resource monitor.
```

```suggestion
    # -- The address of an external resource plugin service, overriding the default resource computation logic based on the percentage of available resources. Leave it empty to use the standard local resource monitor.
```

I would suggest to also add a reference to the repository with the resource plugins.